app/models/event.rb

```diff
@@ -0,0 +1,7 @@
+class Event < ApplicationRecord
+  validates :title , presence:true
+  validates :description , presence:true
```

Comment:
Consider adding a blank line before the first validation for better readability.